src/pandablocks/responses.py

```diff
@@ -266,6 +266,9 @@ class StartData(Data):
     process: str
     format: str
     sample_bytes: int
+    arm_time: str
+    start_time: str
+    hw_time_offset_ns: str
```

Comment:
```suggestion
    hw_time_offset_ns: int
```

Reply:
A different proposed change for the same code is already committed.